### ISSUE TITLE
您好, 根据您的博客内容部署成功了,但是web页面无法连接.

### ISSUE THREAD

**Issue by Dajen-Z** [v3ucn/CosyVoice_For_Windows#25]
(cosyvoice) C:\Tools\CosyVoice_For_Windows>python webui.py --port 9886 --model_dir pretrained_models/CosyVoice-300M
C:\Tools\CosyVoice_For_Windows\cosyvoice\cli\cosyvoice.py:17: UserWarning: torchaudio._backend.set_audio_backend has been deprecated. With dispatcher enabled, this function is no-op. You can remove the function call.
  torchaudio.set_audio_backend('soundfile')
2024-08-23 20:36:06,590 - modelscope - INFO - PyTorch version 2.4.0+cu121 Found.
2024-08-23 20:36:06,590 - modelscope - INFO - Loading ast index from C:\Users\dajen\.cache\modelscope\ast_indexer
2024-08-23 20:36:06,628 - modelscope - INFO - Loading done! Current index file version is 1.15.0, with md5 d6df55a0d8a6452df64900b4f71ce64c and a total number of 980 components indexed
C:\ProgramData\anaconda3\envs\cosyvoice\Lib\site-packages\torch\_jit_internal.py:745: FutureWarning: ignore(True) has been deprecated. TorchScript will now drop the function call on compilation. Use torch.jit.unused now. {}
  warnings.warn(
C:\ProgramData\anaconda3\envs\cosyvoice\Lib\site-packages\diffusers\models\lora.py:393: FutureWarning: `LoRACompatibleLinear` is deprecated and will be removed in version 1.0.0. Use of `LoRACompatibleLinear` is deprecated. Please switch to PEFT backend by installing PEFT: `pip install peft`.
  deprecate("LoRACompatibleLinear", "1.0.0", deprecation_message)
2024-08-23 20:36:10,691 INFO input frame rate=50
C:\ProgramData\anaconda3\envs\cosyvoice\Lib\site-packages\torch\nn\utils\weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
C:\Tools\CosyVoice_For_Windows\cosyvoice\dataset\processor.py:24: UserWarning: torchaudio._backend.set_audio_backend has been deprecated. With dispatcher enabled, this function is no-op. You can remove the function call.
  torchaudio.set_audio_backend('soundfile')
  
  根据您这个教程https://v3u.cn/a_id_362来安装的. 请问原因出在哪?谢谢
